### PR TITLE
Latest updates

### LINKED ISSUE #7: Home Page - Latest Updates
https://www.figma.com/file/KSkquGGDY2MQGrXsTFaAaL/Untitled?node-id=7%3A3

### PR DESCRIPTION
**Pull request by AAYUSHAIRAN**
closes #7

### CODE REVIEW

File: src/components/Home/components/latest_updates/latest_updates.js [hunk 1]
```diff
@@ -1,5 +1,93 @@
 import React from "react";
+<<<<<<< HEAD
```
Comment (parth-paradkar) [inline src/components/Home/components/latest_updates/latest_updates.js]: This seems to be some leftover code from a rebase. Please remove it and push.

**Review by parth-paradkar — CHANGES_REQUESTED** [review]

File: src/components/Home/components/latest_updates/latest_updates.js [hunk 2]
```diff
@@ -1,5 +1,93 @@
...
+          })}
+        </div>
+      </div>
+    </div>
+  );
+=======
 
 export default function LatestUpdates() {
   return <div></div>;
+>>>>>>> adeb85b0f6af6d0ab6c5fc24c801ebc0a33f2b53
```
Comment (parth-paradkar) [inline src/components/Home/components/latest_updates/latest_updates.js]: This too.

File: src/components/Home/components/latest_updates/latest_updates.js (hunk 1 above)
Comment (AAYUSHAIRAN) [reply src/components/Home/components/latest_updates/latest_updates.js]: okkk

File: src/components/Home/components/latest_updates/latest_updates.js [hunk 3]
```diff
@@ -85,9 +84,3 @@ export default function LatestUpdates() {
       </div>
     </div>
   );
-=======
-
-export default function LatestUpdates() {
-  return <div></div>;
->>>>>>> adeb85b0f6af6d0ab6c5fc24c801ebc0a33f2b53
-}
```
Comment (parth-paradkar) [inline src/components/Home/components/latest_updates/latest_updates.js:5]: Shouldn't have removed this closing brace I think.

File: src/components/Home/components/latest_updates/latest_updates.js [hunk 4]
```diff
@@ -1,5 +1,87 @@
...
+        style={{
+          fontFamily: "sans-serif",
+          fontStyle: "normal",
+          fontWeight: "600",
+          fontSize: "60px",
+          textAlign: "center",
+          color: "#FFFFFF",
+          letterSpacing: "4px",
+          padding: "60px",
+        }}
```
Comment (parth-paradkar) [inline src/components/Home/components/latest_updates/latest_updates.js]: Shift this to the CSS file

File: src/components/Home/components/latest_updates/latest_updates.js [hunk 5]
```diff
@@ -1,5 +1,87 @@
...
+            <h2>{list[index - 1].head}</h2>
+            <h5 style={{ margin: "40px" }}>{list[index - 1].text}</h5>
+            <p>{list[index - 1].subtext}</p>
+          </div>
+        </div>
+        <div
+          className="latestUpdatesFotter"
+          style={{
+            marginTop: "30px",
+          }}
```
Comment (parth-paradkar) [inline src/components/Home/components/latest_updates/latest_updates.js]: Move inline style to file.

File: src/components/Home/components/latest_updates/latest_updates.js [hunk 6]
```diff
@@ -1,5 +1,87 @@
...
+            const { id, link } = Index;
+            console.log(id, index);
+            return (
+              <div
+                key={id}
+                className={id == index ? "active" : "not_active"}
+                style={{
+                  backgroundImage: `url(${link})`,
+                  backgroundSize: "cover",
+                }}
```
Comment (parth-paradkar) [inline src/components/Home/components/latest_updates/latest_updates.js:48]: Inline style

**Review by parth-paradkar — CHANGES_REQUESTED** [review]
Inline styles are not preferred, move it to the CSS file

File: src/components/Home/components/latest_updates/latest_updates.js (hunk 6 above)
Comment (AAYUSHAIRAN) [reply src/components/Home/components/latest_updates/latest_updates.js:48]: ye dynamic hai
isko inline hi rkna hogaa
props mai paas ho rha

Comment (parth-paradkar) [reply src/components/Home/components/latest_updates/latest_updates.js:48]: Ok peace

**Review by parth-paradkar — APPROVED** [review]
LGTM

**Review by parth-paradkar — APPROVED** [review]
Resolved conflicts, merging.

**Pull request merged**